docs/pages/api/events.md

```diff
@@ -0,0 +1,87 @@
+---
+description: WebSocket events
+---
+
+# Events
+
+LiveCompositor is using WebSocket connection to send events to the connected clients. Supported events are listed below.
+
+### `VIDEO_INPUT_DELIVERED`
+
+```typescript
+type Event = {
+  type: "VIDEO_INPUT_DELIVERED";
+  input_id: string;
+}
+```
+
+The compositor received the input, and the first frames of that input are ready to be used. If you want to ensure that some inputs are ready before you send the [`start`](./routes.md#start) request, you can wait for those events for specific inputs.
+
+### `VIDEO_INPUT_PLAYING`
+
+```typescript
+type Event = {
+  type: "VIDEO_INPUT_PLAYING";
+  input_id: string;
+}
+```
+
+The compositor received the input and is using the first frame for rendering. This event will not be sent before the [`start`](./routes.md#start) request.
+
+This event is usually sent at the same time as `VIDEO_INPUT_DELIVERED` except for 2 cases:
```

Comment:
Technically no. What about buffering?

Reply:
It's exactly like that both events are sent after buffering. Buffering is an implementation detail, mentioning it here would be only confusing.